system-addon/common/Reducers.jsm

```diff
@@ -95,7 +92,7 @@ function TopSites(prevState = INITIAL_STATE.TopSites, action) {
       }
       return Object.assign({}, prevState, {initialized: true, rows: action.data});
     case at.TOP_SITES_EDIT:
-      return Object.assign({}, prevState, {editForm: {index: action.data.index}});
+      return Object.assign({}, prevState, {editForm: {visible: true, index: action.data.index}});
```

Comment:
`visible` shouldn't be necessary as we should only be showing the form if `editForm` is not `null`?